identity/backend/src/main/resources/schema.sql

```diff
@@ -50,3 +52,26 @@ CREATE TABLE IF NOT EXISTS group_members (
 CREATE TABLE IF NOT EXISTS profiles(
     id bigint primary key, email varchar(100) unique,
     constraint fk_profile_users foreign key(id) references users(id) ON DELETE CASCADE);
+
+-- authority == role
+CREATE TABLE IF NOT EXISTS roles(
+    authority varchar(50) not null primary key,
+    description varchar(500)
+);
+
+CREATE TABLE IF NOT EXISTS permissions(
+    id bigint generated by default as identity(start with 1) primary key,
+    definition varchar(50) not null unique,
+    description varchar(50) not null
+);
```

Comment:
We don't need this table, right?

Reply:
absolutely true! I wonder if I removed it in another branch because I remember I did that somewhere ... thanks for spotting it here!